### PR TITLE
feat: add models and gems

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/db/migrate/20190327073533_create_blocks.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      t.binary :parent_hash
+      t.jsonb :seal
+      t.integer :timestamp
+      t.binary :txs_commit
+      t.binary :txs_proposal
+      t.integer :uncles_count
+      t.binary :uncles_hash
+      t.string :uncle_block_hashes
+      t.integer :version
+      t.binary :proposal_transactions
```
Comment (classicalliu) [inline server/db/migrate/20190327073533_create_blocks.rb]: Only one or an array ?

**Comment by duanyytop** [comment]
We should put pr link to `ckb-explorer-internal` because pr is public and `ckb-explorer-internal` is private.

We should promise that developers out of team could visit any link here.

File: server/db/migrate/20190327073533_create_blocks.rb (hunk 1 above)
Comment (shaojunda) [reply server/db/migrate/20190327073533_create_blocks.rb]: it is an array, but I want pack them into a binary sequence

Comment (classicalliu) [reply server/db/migrate/20190327073533_create_blocks.rb]: Ok

**Pull request merged**